### PR TITLE
Added podspec

### PR DESCRIPTION
**Pull request by poyannabati**
Using #308 as a reference, I added a working podspec once more. 

@ivpusic; I saw you mentioned that people were confused on how to use it. I think we could reorganize the docs to make it clearer. The docs need to be updated on how to use this as well. Let me know if you want me to do this.

### CODE REVIEW

**Comment by ivpusic** [comment]
I guess this wont work now for people who dont use cococaposa because you change header import.

File: ios/ImageCropPicker.h [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #import "QBImagePicker.h"
 #import "RSKImageCropper.h"
 #else
```
Comment (chaitanya0bhagvan) [inline ios/ImageCropPicker.h]: To handle cases where this library is not installed using cocoapods we should include another conditon

```
#elseif __has_include("QBImagePicker/QBImagePicker.h")
#import "QBImagePicker/QBImagePicker.h"
#import <RSKImageCropper/RSKImageCropper.h>

```

**Comment by ivpusic** [comment]
A lot of people were having issues with podspec, so I removed it completely and added installation instructions how to install a library without it.

**Comment by chaitanya0bhagvan** [comment]
Do you see any issue in retaining the podspec file and the changes to the
header file?

**Comment by ivpusic** [comment]
yes...it was breaking the installation. refer to https://github.com/ivpusic/react-native-image-crop-picker/issues/421 and https://github.com/ivpusic/react-native-image-crop-picker/issues/414.

I tried it by myself and was also getting compilation issues.

**Comment by chaitanya0bhagvan** [comment]
I was looking at those issues and I realised the read me I created assumed
that users who would use podspec had installed react native using podspec
too.

I should have been very clear that users should link this library using
react-native link if they are not using podspec to install react native.

You can confirm that retaining the header file changes and podspec won't
cause any build issues if you link this library using react-native link.

Podspec is for those developers who use pods to manage all their dependency.


On 25-Aug-2017 2:04 AM, "Ivan Pusic" <notifications@github.com> wrote:

yes...it was breaking the installation. refer to #421
<https://github.com/ivpusic/react-native-image-crop-picker/issues/421> and
#414 <https://github.com/ivpusic/react-native-image-crop-picker/issues/414>.

I tried it by myself and was also getting compilation issues.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/ivpusic/react-native-image-crop-picker/pull/372#issuecomment-324749683>,
or mute the thread

**Pull request closed without merging**